File: Alyona and Numbers.cpp
#include<bits/stdc++.h>
#define ll long long int
using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(0);
   
    int a,b; cin>>a>>b;

    int small = min(a,b);
    int big = max(a,b);
    ll count = 0;

    for(int i=1; i<=small; i++)
    {
        count += ( ( i + big ) / 5 ) - (i / 5);
    }

    cout<<count<<"\n";
}

File: Alyona and Numbers.py
a, b = map(int, input().split())
 
small = min(a, b)
big = max(a, b)
count = 0
 
for i in range(1, small+1):
    count = count + ((i + big)//5) - (i//5)
 
print(f"{count}")
